src/modules/link-tooltip.coffee

```diff
@@ -82,6 +84,7 @@ class LinkTooltip extends Tooltip
         @textbox.setSelectionRange(0, url.length)
       )
     else
+      dom(@container).addClass('open')
```

Comment:
Oh, I misunderstood. I thought `open` and `editing` were intended to happen at the same time. The three states being:

- `class=""`
- `class="open"`
- `class="open editing"`